bin/node/src/chain_spec.rs

```diff
@@ -278,12 +265,17 @@ fn genesis(
     wasm_binary: &[u8],
     authorities: Vec<AuthorityKeys>,
     root_key: AccountId,
```

Comment:
Can 'root account/key' have different meaning than a 'sudo account/key'? If not, then I would definitely rename this argument, since it introduces some confusion (at least for me :P)

Reply:
makes sense, done